scripts/devtools/tpl-manager.py

```diff
@@ -170,27 +175,25 @@ def build_deps(args):
   # Loop through the specs we want TPLs for and build/install/get them as necessary.
   if not args.init_only:
       for s in spec_list:
-
         print("** Building TPL's and generating host-config for {0}%{1} ...".format(package_name,s))
         os.environ["SPEC"] = s
         os.environ["LC_ALL"] = "en_US.UTF-8"
 
         if not args.no_spec:
-            if sexe("{0} spec --fresh -I {1}@develop%{2}".format(spack_cmd, package_name, s), echo=True) : sys.exit(1)
-
-        # Install only the dependencies for Spheral
-        if sexe("{0} install --fail-fast --fresh --deprecated --only dependencies {2}@develop%{3} 2>&1 | tee -a \"dev-build-{3}-out.txt\"".format(spack_cmd, os.getcwd(), package_name, s), echo=True) : sys.exit(1)
+          sexe("{0} spec --fresh -IL {1}@develop%{2} 2>&1 | tee -a \"spec-info-{2}-out.txt\"".format(spack_cmd, package_name, s))
 
-        # Using dev-build we can have spack generate an init-config with the local source files for spheral.
-        if sexe("{0} dev-build --fresh --deprecated -u initconfig {2}@develop%{3} 2>&1 | tee -a \"dev-build-{3}-out.txt\"".format(spack_cmd, os.getcwd(), package_name, s), echo=True) : sys.exit(1)
+        # Install only the dependencies for Spheral and create CMake configure file
+        sexe("{0} install --fail-fast --fresh -u initconfig {1}@develop%{2} dev_path=./ 2>&1 | tee -a \"tpl-build-{2}-out.txt\"".format(spack_cmd, package_name, s))
```

Comment:
Does `dev_path` have any other side effects, or alter paths for how the dependencies are being built? Or does it only effect the spec/package you are targeting (so spheral only in this case)?

Reply:
The `dev_path` stuff is essentially what the dev-build command is doing under the hood. It shouldn't do anything different regarding building the TPLs except it does provide a symbolic link to the spheral build directory for some reason.